ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/solution_expose.py

```diff
@@ -95,8 +146,9 @@ def chat(bot):
         entrypoint=entrypoint,
         secret_env=secret_env,
     )
+
     # create proxy
-    j.sal.zosv2.gateway.tcp_proxy_reverse(reservation, gateway_id, domain, user_form_data["secret"])
+    j.sal.zosv2.gateway.tcp_proxy_reverse(reservation, domain_gateway.node_id, domain, user_form_data["secret"])
```

Comment:
where is the part you add the subdomain to the reservation incase it's not a custom domain?